sycl/include/CL/sycl/queue.hpp

```diff
@@ -79,7 +79,7 @@ class queue {
   /// \param SyclDevice is an instance of SYCL device.
   /// \param AsyncHandler is a SYCL asynchronous exception handler.
   /// \param PropList is a list of properties for queue construction.
-  queue(const device &SyclDevice, const async_handler &AsyncHandler,
+  explicit queue(const device &SyclDevice, const async_handler &AsyncHandler,
         const property_list &PropList = {});
```

Comment:
```suggestion
                  const property_list &PropList = {});
```
This seems to fix the problem.

Reply:
Oh. Thanks. I'm surprised that causes an issue since I technically did not touch that line in this patch. The script I use clang-format-diff.py only formats patch diff without surrounding context IIRC.

Reply:
I always just build the clang-format target too.  There's also a nice emacs plugin that can be hot-keyed. :)

Reply:
> Sorry. ClangFormat clang-format-diff.py script was erroring out when I tried running it on this patch and so I just corrected the issues manually. I see now there are more issues. I'll figure out what's wrong with the script and run the actual tool now.

If you also build clang-format along with sycl-toolchain, you'll generate the clang-format binary.

Reply:
Yes thank you. The script was throwing an error because I had missed building clang-format. I ran it now and it caught this issue. I'm pushing the amended patch now.